icons/forgejo/forgejo-original-wordmark.svg

```diff
@@ -0,0 +1,110 @@
+       id="path1099"
+       style="fill:#ff5500;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1;stroke-dasharray:none;stroke-opacity:1" />
+    <path
+       d="m 601.862,285.9567 c -4.9866,0 -9.7466,-0.6044 -14.28,-1.8133 -4.5333,-1.2089 -8.311,-3.0978 -11.3333,-5.6667 l 6.3467,-14.28 c 4.6844,3.7778 10.7289,5.6667 18.1333,5.6667 6.0445,0 10.6534,-1.8134 13.8267,-5.44 3.3245,-3.6267 4.9867,-9.0667 4.9867,-16.32 V 141.1698 h 19.04 v 106.9335 c 0,11.6356 -3.1733,20.8534 -9.52,27.6534 -6.3466,6.8 -15.4133,10.2 -27.2,10.2 z"
+       id="path1103"
+       style="fill:#ff5500;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1;stroke-dasharray:none;stroke-opacity:1" />
+    <path
+       d="m 717.3326,241.9833 q -17.68,0 -31.7333,-7.7066 -13.8267,-7.9334 -21.9867,-21.76 -7.9334,-13.8268 -7.9334,-31.5068 0,-17.9067 7.9333,-31.5067 8.16,-13.8267 21.9867,-21.5334 13.8267,-7.9333 31.7334,-7.9333 17.68,0 31.5067,7.9333 14.0534,7.7067 21.9867,21.5334 7.9334,13.6 7.9334,31.5067 0,17.68 -7.9334,31.5067 -7.9333,13.8267 -21.9867,21.76 -13.8266,7.7067 -31.5067,7.7067 z m 0,-16.5467 q 12.24,0 21.76,-5.44 9.52,-5.6666 14.96,-15.64 5.6668,-10.2 5.6668,-23.3467 0,-13.3733 -5.6667,-23.3467 -5.44,-9.9733 -14.96,-15.4133 -9.52,-5.44 -21.76,-5.44 -12.24,0 -21.7601,5.44 -9.52,5.44 -15.1867,15.4133 -5.44,9.9734 -5.44,23.3467 0,13.1467 5.44,23.3467 5.6667,9.9734 15.1867,15.64 9.52,5.44 21.76,5.44 z"
+       id="path1105"
+       style="fill:#ff5500;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1;stroke-dasharray:none;stroke-opacity:1" />
+    <path
+       d="m 547.1384,242.2469 q -18.8133,0 -33.32,-7.7067 -14.28,-7.9334 -22.44,-21.76 -7.9334,-13.8268 -7.9334,-31.5068 0,-17.68 7.7067,-31.5067 7.7066,-13.8267 21.08,-21.5333 13.6,-7.9334 30.8267,-7.9334 16.32,0 29.24,7.2534 12.92,7.2533 20.6268,20.4 7.9333,12.92 8.3866,30.3734 l -101.7734,19.9466 -0.9067,-13.8266 91.5735,-17.68 -7.0267,9.9733 q 0,-11.3334 -5.2134,-20.4 -4.9866,-9.0667 -14.0533,-14.28 -9.0667,-5.44 -20.8534,-5.44 -12.0133,0 -21.5333,5.44 -9.2934,5.44 -14.5067,15.1866 -5.2134,9.7467 -5.2134,22.8934 0,13.8267 5.6667,24.2534 5.8933,10.2 16.0934,15.8666 10.4266,5.44 24.0267,5.44 10.88,0 19.9467,-3.6266 9.0666,-3.8534 15.64,-11.56 l 10.4267,12.24 q -7.7067,9.52 -19.72,14.5067 -12.0134,4.9866 -26.7468,4.9867 z"
+       id="path11315"
+       style="font-size:226.667px;line-height:1.25;font-family:Montserrat;-inkscape-font-specification:'Montserrat, @wght=451';font-feature-settings:'ss01';font-variation-settings:'wght' 451;fill:#ff5500;fill-opacity:1;stroke:none;stroke-width:20" />
+  </g>
+  <ellipse
+     cx="103.04945"
+     cy="43.806793"
+     class="orange"
+     id="circle15505-0"
+     style="fill:none;stroke:#ff5500;stroke-width:3.75877;stroke-opacity:1"
+     rx="2.8276513"
+     ry="7.8070116" />
+</svg>
```

Comment:
What is wrong with this image
- Aspect ratio is wrong, logo is squished
- Image contains Inkscape metadata and is not optimized

what did i change here:
- Ungroup all paths
- convert all strokes to paths
- group all elements by color (2 path elements left)
- make sure the logo is square and centered
- resized the image to 128x128 using iloveimg
- optimize the image using vecta

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M19.73 40.855c-1.645 0-3.09.902-3.867 2.238h-1.852a11.81 11.81 0 0 0-11.773 11.77v19.07h4.477v-19.07a7.26 7.26 0 0 1 7.297-7.293h1.852a4.49 4.49 0 0 0 3.867 2.238 4.5 4.5 0 0 0 4.477-4.477 4.5 4.5 0 0 0-4.477-4.477zm0 2.652c1.023 0 1.824.801 1.824 1.824s-.801 1.824-1.824 1.824-1.824-.801-1.824-1.824.801-1.824 1.824-1.824zm83.207 10.211c-2.195 0-4.004 1.809-4.004 4.004a4.03 4.03 0 0 0 4.004 4.008c2.199 0 4.004-1.809 4.004-4.008s-1.805-4.004-4.004-4.004zm0 2.289a1.7 1.7 0 1 1 0 3.434c-.961 0-1.715-.754-1.715-1.719s.754-1.715 1.715-1.715zM32.359 59.57c-1.977 0-3.73.438-5.262 1.316a9.6 9.6 0 0 0-3.645 3.57c-.875 1.504-1.316 3.246-1.316 5.223 0 1.957.441 3.695 1.316 5.227.902 1.527 2.117 2.73 3.645 3.605 1.555.852 3.309 1.277 5.262 1.277s3.695-.426 5.227-1.277a9.53 9.53 0 0 0 3.645-3.605c.875-1.531 1.316-3.27 1.316-5.227 0-1.977-.441-3.719-1.316-5.223-.879-1.527-2.094-2.719-3.645-3.57-1.531-.879-3.27-1.316-5.227-1.316zm23.191 0c-1.93 0-3.547.391-4.848 1.164a6.17 6.17 0 0 0-2.48 2.602v-3.578h-3.008v19.844h3.156v-9.996c0-2.254.578-3.984 1.73-5.187 1.152-1.227 2.719-1.84 4.695-1.84.125 0 .25.012.379.035.125 0 .25.016.375.039zm10.652 0c-1.906 0-3.621.402-5.148 1.203-1.504.801-2.695 1.93-3.574 3.383-.875 1.43-1.312 3.094-1.312 5s.438 3.582 1.313 5.035a9.36 9.36 0 0 0 3.574 3.418c1.527.801 3.242 1.203 5.148 1.203 1.727 0 3.281-.363 4.66-1.09.84-.437 1.578-1 2.215-1.691l-.098 1.355c-.172 2.348-.578 4.094-1.73 5.223-1.152 1.152-2.945 1.727-5.375 1.727a13.09 13.09 0 0 1-4.281-.711c-1.355-.477-2.496-1.129-3.422-1.957l-1.578 2.445c1.078.949 2.457 1.676 4.133 2.18 1.703.523 3.457.789 5.262.789 3.332 0 5.852-.816 7.555-2.445 1.703-1.602 2.555-4.133 2.555-7.59 0-4.766.625-10.625-1.227-13.191-.828-1.43-1.941-2.492-3.348-3.195-1.375-.727-3.594-1.09-5.32-1.09zm51.547 0c-1.98 0-3.734.438-5.262 1.316a9.6 9.6 0 0 0-3.645 3.57c-.879 1.504-1.316 3.246-1.316 5.223 0 1.957.438 3.695 1.316 5.227.902 1.527 2.117 2.73 3.645 3.605 1.555.852 3.309 1.277 5.262 1.277s3.695-.426 5.223-1.277c1.555-.875 2.77-2.078 3.648-3.605s1.313-3.27 1.313-5.227c0-1.977-.437-3.719-1.312-5.223-.879-1.527-2.094-2.719-3.648-3.57-1.527-.879-3.27-1.316-5.223-1.316zm-28.895.043c-1.906 0-3.609.441-5.109 1.316-1.48.852-2.645 2.043-3.496 3.57s-1.277 3.27-1.277 5.223.438 3.695 1.313 5.227c.902 1.527 2.145 2.73 3.723 3.605 1.602.852 3.445 1.277 5.523 1.277 1.629 0 3.105-.273 4.434-.824s2.418-1.355 3.27-2.406l-1.727-2.027c-.727.852-1.59 1.488-2.594 1.914-1.004.402-2.105.602-3.309.602-1.5 0-2.828-.301-3.98-.902a6.91 6.91 0 0 1-2.672-2.629c-.199-.371-.367-.762-.504-1.176l16.066-3.148c-.051-1.93-.516-3.605-1.391-5.035-.852-1.453-1.992-2.582-3.422-3.383s-3.043-1.203-4.848-1.203zm0 2.707c1.301 0 2.457.301 3.457.902a6.09 6.09 0 0 1 2.332 2.367 6.39 6.39 0 0 1 .73 1.98l-13.34 2.574a10.36 10.36 0 0 1-.02-.609c0-1.453.289-2.719.863-3.793s1.379-1.918 2.406-2.52c1.051-.602 2.242-.902 3.57-.902zm-56.496.031c1.355 0 2.559.301 3.609.902s1.879 1.453 2.48 2.555c.625 1.105.938 2.395.938 3.871 0 1.453-.312 2.746-.937 3.871-.602 1.102-1.43 1.969-2.48 2.594-1.051.602-2.254.902-3.609.902s-2.555-.301-3.605-.902c-1.055-.625-1.891-1.492-2.52-2.594-.602-1.125-.902-2.418-.902-3.871 0-1.477.301-2.766.902-3.871.629-1.102 1.465-1.953 2.52-2.555s2.254-.902 3.605-.902zm33.813 0c1.402 0 2.641.289 3.719.863 1.078.555 1.918 1.344 2.52 2.371.625 1.023.938 2.215.938 3.57s-.312 2.555-.937 3.605c-.602 1.027-1.441 1.828-2.52 2.406s-2.316.863-3.719.863c-1.379 0-2.605-.289-3.684-.863s-1.93-1.379-2.555-2.406c-.602-1.051-.902-2.254-.902-3.605s.301-2.547.902-3.57c.625-1.027 1.477-1.816 2.555-2.371 1.078-.574 2.305-.863 3.684-.863zm51.578 0c1.352 0 2.555.301 3.609.902s1.879 1.453 2.48 2.555c.625 1.105.938 2.395.938 3.871 0 1.453-.312 2.746-.937 3.871-.602 1.102-1.43 1.969-2.48 2.594-1.055.602-2.258.902-3.609.902s-2.555-.301-3.609-.902c-1.051-.625-1.891-1.492-2.516-2.594-.602-1.125-.902-2.418-.902-3.871 0-1.477.301-2.766.902-3.871.625-1.102 1.465-1.953 2.516-2.555s2.258-.902 3.609-.902zm-16.215.723v17.73c0 1.203-.273 2.105-.824 2.707-.527.602-1.293.902-2.293.902-1.23 0-2.23-.312-3.008-.941l-1.051 2.367c.5.426 1.125.738 1.879.941s1.539.301 2.367.301c1.953 0 3.457-.566 4.508-1.691s1.578-2.656 1.578-4.586v-17.73zm0 0" fill="#ff5500"/><path d="M19.73 53.246a4.49 4.49 0 0 0-3.867 2.238h-1.852c-6.477 0-11.773 5.301-11.773 11.773v4.711C.906 72.75 0 74.195 0 75.84a4.5 4.5 0 0 0 4.477 4.477 4.5 4.5 0 0 0 4.477-4.477c0-1.645-.902-3.09-2.238-3.871v-4.711a7.26 7.26 0 0 1 7.297-7.293h1.852a4.49 4.49 0 0 0 3.867 2.238c2.457 0 4.477-2.023 4.477-4.48a4.5 4.5 0 0 0-4.477-4.477zm0 2.656c1.023 0 1.824.801 1.824 1.82s-.801 1.824-1.824 1.824-1.824-.801-1.824-1.824.801-1.82 1.824-1.82zM4.477 74.016c1.023 0 1.824.801 1.824 1.824s-.801 1.824-1.824 1.824-1.824-.801-1.824-1.824.801-1.824 1.824-1.824zm0 0" fill="#cc0000"/></svg>

```